plugins/chat/test/javascripts/acceptance/hashtag-css-generator-test.js

```diff
@@ -62,16 +62,4 @@ acceptance("Chat | Hashtag CSS Generator", function (needs) {
       })
     );
   });
-
-  test("hashtag CSS classes are generated", async function (assert) {
```

Comment:
This is a duplicate test of:

app/assets/javascripts/discourse/tests/acceptance/hashtag-css-generator-test.js:22